etn-ccis/blui-react-workflows
# Feature/blui 4130 verify code screen base

Fixes BLUI-4130



#### Changes proposed in this Pull Request:

- Add Verify code screen base
-
-



#### Screenshots / Screen Recording (if applicable)

![image](https://github.com/etn-ccis/blui-react-workflows/assets/119693939/bb5f10fb-1c7e-4def-8fd7-b5c07231e971)
![image](https://github.com/etn-ccis/blui-react-workflows/assets/119693939/a0c7369c-5fea-4b0e-95e1-8663445a3181)




#### To Test:

- yarn start:example2
- select test verify code screen
- open dev tools> console verify calls on validator and resend



#### Any specific feedback you are looking for?

-
